src/en/users-models.md

```diff
@@ -112,6 +112,50 @@ bob                   login      1 hour ago    58 minutes ago
 jim                   add-model  2016-11-14    58 minutes ago
 ```
 
+### Controller access for external users
+
+It is possible to give a user access to a controller without creating a local
+account for them. To do this, these criteria must first be met:
+
+- The user must already have an account on an external identity manager,
+  such as the Ubuntu SSO
+- The controller must have been created (bootstrapped) using the identity
+  configuration option, like here where we use the URL for the Ubuntu SSO
+  and Juju: `--config identity-url=https://api.jujucharms.com/identity`
+- The user must first log in to http://jujucharms.com at least once before
```

Comment:
This is true atm, however, we'll deprecate this requirement in 2 weeks so it should be revisited later. Added a todo item on our side to ping you when we remove this requirement.